demo/actions.py

```diff
@@ -662,12 +668,13 @@ def name(self) -> Text:
         return "action_next_step"
 
     def run(self, dispatcher, tracker, domain) -> List[EventType]:
-        step = int(tracker.get_slot("step")) + 1
+        if tracker.get_slot("step"):
```

Comment:
What should this do if there is no step though? Certainly not nothing -- maybe step 1?

Reply:
Good idea!

When this happens, I am now triggering the `get_started_step1` intent, and set the slot `step=1`. Works nice.

You can test this fix by starting `rasa shell`, and passing `/next_step`